.github/workflows/tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: sdk-tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: 'MobileCoin Network'
+        required: true
+        default: 'MOBILE_DEV'
+        type: choice
+        options:
+          - 'MOBILE_DEV'
+          - 'ALPHA'
+          - 'TEST_NET'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: GCP Service Account
+        run: |
+          echo '${{ secrets.GCP_SERVICE_KEY }}' > "service-key.json"
+
+      - name: Provide Dev Credentials
+        env:
+          TEST_DEV_USER: ${{ secrets.TEST_DEV_USER }}
+          TEST_DEV_PASSWORD: ${{ secrets.TEST_DEV_PASSWORD }}
+        run: |
+          sed -i 's/REPLACE_TEST_DEV_USER_STRING/'"${TEST_DEV_USER}"'/g' android-sdk/src/androidTest/java/com/mobilecoin/lib/TestFogConfig.java && \
+          sed -i 's/REPLACE_TEST_DEV_PASSWORD_STRING/'"${TEST_DEV_PASSWORD}"'/g' android-sdk/src/androidTest/java/com/mobilecoin/lib/TestFogConfig.java
```

Comment:
This works but is a bit messy, just put the regex in double quotes.

Reply:
Cleaned up the commands. Also adjusted RegEx for the network as the value being checked was changed in a recent PR so that `sed` command would have been broken and is the setting pretty much for this whole PR.